File: app/src/main/java/com/xia/banner/option/text/NoticeHolderCreator.java
package com.xia.banner.option.text;

import android.view.View;

import com.xia.banner.R;
import com.xia.flybanner.holder.FBHolder;
import com.xia.flybanner.holder.FBViewHolderCreator;

/**
 * @author weixia
 * @date 2019/4/16.
 */
public class NoticeHolderCreator implements FBViewHolderCreator {

    @Override
    public FBHolder createHolder(View itemView) {
        return new NoticeHolder(itemView);
    }

    @Override
    public int getLayoutId() {
        return R.layout.item_notice_text;
    }
}

File: app/src/main/java/com/xia/banner/option/text/NoticeHolderCreator.kt
package com.xia.banner.option.text

import android.view.View

import com.xia.banner.R
import com.xia.flybanner.holder.FBHolder
import com.xia.flybanner.holder.FBViewHolderCreator

/**
 * @author weixia
 * @date 2019/4/16.
 */
class NoticeHolderCreator : FBViewHolderCreator {

    override fun createHolder(itemView: View): FBHolder<*> {
        return NoticeHolder(itemView)
    }

    override fun getLayoutId(): Int {
        return R.layout.item_notice_text
    }
}
